spec/requests/clicked_serp_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'spec_helper'
+
+describe 'Clicked' do
```

Comment:
We could DRY up the api vs. non-api request specs by using some shared behavior, i.e. `it_behaves_like 'a click request'`